packages/syft/src/syft/service/settings/settings.py

```diff
@@ -176,22 +225,138 @@ def _repr_html_(self) -> Any:
             """
 
 
-@migrate(ServerSettingsV1, ServerSettings)
+@serializable()
+class ServerSettings(SyftObject):
+    __canonical_name__ = "ServerSettings"
+    __version__ = SYFT_OBJECT_VERSION_3
+    __repr_attrs__ = [
+        "name",
+        "organization",
+        "description",
+        "deployed_on",
+        "signup_enabled",
+        "admin_email",
+    ]
+
+    id: UID
+    name: str = "Server"
+    deployed_on: str
+    organization: str = "OpenMined"
+    verify_key: SyftVerifyKey
+    on_board: bool = True
+    description: str = "This is the default description for a Datasite Server."
+    server_type: ServerType = ServerType.DATASITE
+    signup_enabled: bool
+    admin_email: str
+    server_side_type: ServerSideType = ServerSideType.HIGH_SIDE
+    show_warnings: bool
+    association_request_auto_approval: bool
+    eager_execution_enabled: bool = False
+    default_worker_pool: str = DEFAULT_WORKER_POOL_NAME
+    welcome_markdown: HTMLObject | MarkdownDescription = HTMLObject(
+        text=DEFAULT_WELCOME_MSG
+    )
+    notifications_enabled: bool
+    pwd_token_config: PwdTokenResetConfig = PwdTokenResetConfig()
+
+    def _repr_html_(self) -> Any:
```

Comment:
I think we can remove the `_repr_html_` method from ServerSettingsV2 right?

Reply:
Done! [74f1b9d](https://github.com/OpenMined/PySyft/pull/9052/commits/74f1b9da8124b0468a2c59fca716d17e103f8916)